tests/testthat/test-control_chart.R

```diff
@@ -0,0 +1,69 @@
+context("Checking control_chart")
+
```

Comment:
How do people feel about adding a comment block at the top to describe what's tested? Yannick did this for a couple large test files and it was helpful.

Reply:
I think context identifies the function you're testing and the `test_that` description identifies the aspect of the function you're testing. I'd hope that would be enough, but if a test or setup needs more description, then sure, add some comments.